CHANGELOG.rst

```diff
@@ -18,6 +18,15 @@ Changed
 
   Contributed by @Kami.
 
+* Default nginx config (``conf/nginx/st2.conf``) which is used by the installer and Docker
+  images has been updated to only support TLS v1.2 (support for TLS v1.0 and v1.1 has been
+  removed).
+
+  If someone still needs to support those versions, they can modify the nginx config
+  accordingly. #5183
+
+  Contributed by @Kami and @ashwini.
```

Comment:
It's not only about the TLS change.
Can we include a note about hardening nginx config with extra security headers?

That's where we started.